_posts/2024-12-25-Reflections-as-Parents-and-Teachers-Sugar-at-home-and-in-their-classroom.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: story
+title: "Reflections-as-Parents-and-Teachers-Sugar-at-home-and-in-their-classroom"
```

Comment:
The title should be the same as the title is in the article.

Also dashes are unnecessary. They render in the HTML, which is not what we want.

![Screenshot from 2025-01-04 16-09-39](https://github.com/user-attachments/assets/9a8d6097-110e-4a89-a92f-ec6fe272d8a4)